Dockerfile

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright Contributors to the Egeria project
+
+# This Dockerfile should be run from the parent directory of the egeria-connector-sas-viya directory
+# ie
+# docker -f ./Dockerfile 
+
+FROM odpi/egeria
+
+# Thes are optional tags used to add additional metadata into the docker image
+# These may be supplied by the pipeline in future - until then they will default
+
+ARG version=2.10-SNAPSHOT
```

Comment:
see above - just need this earlier before the from

Reply:
Should this default to "latest" and allow the specific version to come in from the merge.yaml "env.RELEASE_VERSION"?

Reply:
Yes I think that would be reasonable